### DISCUSSION TITLE
Welcome to jrnl Discussions!

### DISCUSSION THREAD

**Discussion by micahellison** [jrnl-org/jrnl#1297] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪. Our [code of conduct](https://github.com/jrnl-org/jrnl/blob/develop/CODE_OF_CONDUCT.md) applies here too.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bduplanty** [comment]
Use jrnl for work and personal and use with imdone.io.  Looking to refine personal recipes to be even more productive!  Thanks for all of the work on this cool tool!

**Comment by minchinweb** [comment]
*jrnl* was likely my [first contribution to open source](https://github.com/jrnl-org/jrnl/commit/e176e1145b40f3e733aa5dbec489548eb871b0a4) back in 2013. I was about that time that I was just starting to pick up Python and *jrnl* in the years since has often served as an example of good Python code to me.

I use *jrnl* to keep my own collection of personal notes and (daily) work notes. I also use it to transform the whole collection, including even more notes kept in DayOne, into something that can be fed into [Pelican](https://github.com/getpelican/pelican), a static site generator.

So glad I found *jrnl* all those years ago!

**Comment by ghost** [comment]
Hey team! hey there!
